File: module-server/src/main/java/de/twomartens/wahlrecht/property/ServiceProperties.java
package de.twomartens.wahlrecht.property;

import io.swagger.v3.oas.annotations.media.Schema;
import java.time.ZoneId;
import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.cloud.context.config.annotation.RefreshScope;
import org.springframework.context.annotation.Configuration;

@Data
@RefreshScope
@Configuration
@ConfigurationProperties(prefix = "de.twomartens.wahlrecht")
@Schema(description = "Properties, to configure this Application")
public class ServiceProperties {

  private ZoneId defaultTimeZone;

  private String greeting;

}

File: module-server/src/main/kotlin/de/twomartens/wahlrecht/property/ServiceProperties.kt
package de.twomartens.wahlrecht.property

import io.swagger.v3.oas.annotations.media.Schema
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.cloud.context.config.annotation.RefreshScope
import java.time.ZoneId

@RefreshScope
@ConfigurationProperties(prefix = "de.twomartens.wahlrecht")
@Schema(description = "Properties, to configure this Application")
open class ServiceProperties {
    lateinit var defaultTimeZone: ZoneId
    lateinit var greeting: String
}
